crates/typst-syntax/src/ast.rs

```diff
@@ -195,6 +195,8 @@ pub enum Expr<'a> {
     For(ForLoop<'a>),
     /// A module import: `import "utils.typ": a, b, c`.
     Import(ModuleImport<'a>),
+    /// A renamed module import (without items): `import "file.typ" as utils`
+    RenamedImport(RenamedModuleImport<'a>),
```

Comment:
Unless there's a good reason, this shouldn't require a separate syntax kind. You can use the normal `ModuleImport`.